connector/cf/cf.go

```diff
@@ -0,0 +1,394 @@
+package cf
```

Comment:
I'd prefer if this would be something like `cloudfoundry`. Cf is too short and not descriptive.